packages/botfuel-dialog/src/config.js

```diff
@@ -26,14 +27,17 @@ const defaultConfig = {
   intentThreshold: 0.8,
   logger: 'info',
   multiIntent: false,
+  conversationDuration: 86400000, // in ms
 };
 
+const whitelist = Object.keys(defaultConfig).concat(['qna', 'spellchecking']);
+
 /**
  * Returns the contents of the bot config file
  * @param {String} configFileName - the bot config file name/path
  * @returns {Object} the contents of the bot config file
  */
-const resolveConfigFile = (configFileName) => {
+const resolveConfigFile = (configFileName = null) => {
```

Comment:
Can you explain why you added `null` as default value?

Reply:
It was a mistake